strength_percentiles.py

```diff
@@ -207,38 +209,40 @@ def populate_database(webpage, database, table):
     connection.close()
 
 ####### Pandas functions
-def find_percentile(dataframe, lifts):
+def find_percentile(lifts_df, user_lifts):
     """
     Finds the percentile of the input squat, bench, deadlift and total numbers
     among the sample of results in the database.
 
     Parameters:
     -----------
-    dataframe: pandas dataframe
+    lifts_df: pandas dataframe
         Dataframe containing squat, bench, deadlift, and/or total numbers
         against which to compare the input numbers.
-    lifts: dictionary with string keys and float (or None) values
+    user_lifts: dictionary with string keys and float (or None) values
         A dictionary that contains keys ('squat', 'bench', 'deadlift', 'total').
         The values are floats representing someone's performance in the lift.
 
     Returns:
     --------
-    percentile: float
-        The user's percentile rank among the lifters in the database in the
-        input categories.
+    percentile: dictionary with string keys string values
+        Dictionary where key is one of "squat", "bench", and "deadlift" and
+        the value is either a string relaying the input lifts percentile
+        among the lifts in the database, or "N/A" if a valid number was not
+        input for the lift.
 
     """
-    percentiles = []
+    percentiles = {SQUAT: None, BENCH: None, DEADLIFT: None}
     for lift in (SQUAT, BENCH, DEADLIFT, TOTAL):
-        if lifts[lift]: # valid entry for lift
-            entered_lift = lifts[lift]
-            all_competitor_lifts = dataframe[lift]
+        if isinstance(user_lifts[lift], numbers.Real): # valid numeric entry for lift
```

Comment:
Could be a neg value too right?
In general for these sorts of things you can usually just assume the input is the right type(as long as you specify the proper types for the inputs in your docstrings). And if the code craps out when someone runs it, well they should have had the proper input type/format then. It's too much work to have to check every possible corner case.